index.html

```diff
@@ -0,0 +1,69 @@
+        <input type="text" name="search_query" id="search" placeholder="Search">
```

Comment:
Please try to use shorter image addresses because it makes your code look nicer and easier to understand.